tests/functional/test_scoping/test_global_import_modules.py

```diff
@@ -151,3 +153,41 @@ def test_globalimports_basic_test_with_distributed_model():
     #################################
     # END
     #################################
+
+
+def test_globalimports_with_project_root_model_parameter():
+    """
+    Basic test for the FQNGlobalRepo.
+    Tests that "project_root" model parameter has an effect.
+    """
+    #################################
+    # META MODEL DEF
+    #################################
+
+    my_meta_model = metamodel_from_file(
+        join(abspath(dirname(__file__)), 'interface_model2',
+             'Interface.tx'))
+    my_meta_model.register_scope_providers(
+        {"*.*": scoping_providers.FQNGlobalRepo(
+            join("model_a", "*.if"))})
+    my_meta_model._tx_model_kwarg_definitions.add_definition(
```

Comment:
This is how a definition can be defined at the current state of implementation. At the moment there is no concept of automatically define parameters of, e.g., scope providers. You need to know that the GlobalRepo evaluates a parameter called "project_root".

However, you could check that the parameter was consumed by a concrete model.